ton-drive-backend/src/ton-storage.service.ts

```diff
@@ -0,0 +1,90 @@
+import { Injectable } from '@nestjs/common';
+import { spawn } from 'node:child_process';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as process from 'process';
+
+export function parseCreateCmdOutput(out: string): string | null {
+  if (out.indexOf('Bag created') == -1) {
+    return null;
+  }
+  const regex = /BagID = (?<bagId>.{64})/;
+  const res = out.match(regex);
+
+  return res ? res.groups['bagId'] : null;
+}
```

Comment:
Why not make `parseCreateCmdOutput()` part of the service class? If you want to hide it in the instance and keep it testable, you can:

1. make method static
2. use [symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) and create private method for usage convenience inside class instance